landoapi/landing_worker.py

```diff
@@ -236,6 +248,10 @@ def run_job(self, job, repo, hgrepo, treestatus, patch_bucket):
                     )
                     hgrepo.apply_patch(patch_buf)
 
+                # Run `hg fix` configured formatters if enabled
+                if repo.formatting:
+                    hgrepo.format()
```

Comment:
I think it would be helpful here to set a `job.is_autoformatted` flag so we can use it to indicate this in the UI. We could also make it more general as a list of flags.

Reply:
> I think it would be helpful here to set a `job.is_autoformatted` flag so we can use it to indicate this in the UI. We could also make it more general as a list of flags.

Right now this code just returns if the autoformatting doesn't fail. We'll need to pass information indicating if the changesets were actually changed by the formatters. I'll add that in the next iteration.

Would adding `job.is_autoformatted` require a change to the DB schema?

Reply:
> Right now this code just returns if the autoformatting doesn't fail. We'll need to pass information indicating if the changesets were actually changed by the formatters. I'll add that in the next iteration.

Sounds good

> Would adding `job.is_autoformatted` require a change to the DB schema?

Yes -- found in `landoapi.models.landing_job`.

Reply:
Still need to address this.